app/admin/page.tsx

```
@@ -111,75 +141,144 @@ export default async function AdminPage() {
+            </h2>
+            <Link
+              href="/admin/members"
+              className="font-sans text-base font-semibold text-brand-primary hover:underline"
+            >
+              View all →
+            </Link>
+          </div>
+          <Card>
+            <CardContent className="pt-6">
+              {pendingError ? (
+                <p className="text-base text-muted-foreground py-3">Couldn't load pending requests.</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)